test/test_runner.c

```diff
@@ -155,7 +155,7 @@ void CB2_TestRunner(void)
         if (gTestRunnerState.test->runner->tearDown)
             gTestRunnerState.test->runner->tearDown(gTestRunnerState.test->data);
 
-        if (!gTestRunnerState.expectLeaks)
+        if (gTestRunnerState.result == gTestRunnerState.expectedResult && !gTestRunnerState.expectLeaks)
```

Comment:
How would you feel about splitting this onto two lines?
```suggestion
        if (gTestRunnerState.result == gTestRunnerState.expectedResult
         && !gTestRunnerState.expectLeaks)
```